pkg/console/subresource/consoleserver/types.go

```diff
@@ -104,10 +104,10 @@ type Session struct {
 
 // Customization holds configuration such as what logo to use.
 type Customization struct {
-	Branding             string `yaml:"branding,omitempty"`
-	DocumentationBaseURL string `yaml:"documentationBaseURL,omitempty"`
-	CustomProductName    string `yaml:"customProductName,omitempty"`
-	CustomLogoFile       string `yaml:"customLogoFile,omitempty"`
+	Branding             string                  `yaml:"branding,omitempty"`
+	DocumentationBaseURL string                  `yaml:"documentationBaseURL,omitempty"`
+	CustomProductName    string                  `yaml:"customProductName,omitempty"`
+	CustomLogos          []operatorv1.CustomLogo `yaml:"customLogos,omitempty"`
```

Comment:
We need to re-evaluate what date/type should be provided to the console server here, because `operatorv1.CustomLogo.Themes.File` is a reference to ConfigMap `configv1.ConfigMapFileReference`. This makes the validation of the CLI `customLogos ` argument problematic as the absolute path of the logo file cant be derived from this reference. 

I think we should create a new type where we would pass the absolute path to the logo file (customLogo volume mount + logo file). Since we would create a new type, I had an idea to normalize the data one step further and do something similar as we do with `--plugins`. Which would look like 
```suggestion
	CustomLogos          map[LogosEnumType]string `yaml:"customLogos,omitempty"`
```
where the enum type would hold one level of all possible logos. Compared to previous option where we pass a big nested json object, this would result in a much cleaner CLI argument - `./bin/bridge --customLogoFiles mastheadDark=/tmp/masthead-dark.png mastheadLight=/tmp/masthead-light.png `

Reply:
This would require a transformation of key strings on both console and operator which might be undesired for us tho.

Reply:
Just wanted to leave the note here after our discussions, so as I understand it we will be splitting my proposed change above into the following format (`./bin/bridge --customLogoFiles Dark=/path/to/hell.png Light=/path/to/heaven.png --customFaviconFiles Default=path/to/almighty.png`). Which in code should be reflected as:
```
type Customization struct {
         ...
	CustomLogoFiles              map[ThemeType]string `yaml:"customLogoFiles,omitempty"`
        CustomFaviconFiles          map[ThemeType]string `yaml:"customFaviconFiles ,omitempty"`
        ...
```
Feel free to correct me @jhadvig

Reply:
I agree with @Mylanos here, lets rather have two separate fields, both in console-config and also as flags. This way it would be easier to work with the SERVER_FLAGS on the frontend and also the overall UX for developemnt would be better since we wont need to pass an JSON.

Reply:
IMO, we should keep the format as-is for the console config, then parse and convert it to whatever format we want in the CLI flags and window.SERVER_FLAGS on the bridge side. I think changing the format on the console operator side introduces unnecessary complexity and potential confusion.